<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalysisForm</class>
 <widget class="QWidget" name="AnalysisForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1595</width>
    <height>1044</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="gbExecution">
     <property name="title">
      <string>Execution</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Working directory:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leWorkingDirectory">
          <property name="toolTip">
           <string>Given path has to exist. If left empty: unique directory will be created.</string>
          </property>
          <property name="placeholderText">
           <string>(auto create)</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSelectWorkingDirectory">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbRemoveWorkingDirectory">
          <property name="text">
           <string>remove after analysis is finished</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="gbExecuteOnRemoteHost">
        <property name="title">
         <string>Execute on remote host</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="lbRR1">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>run on:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="ddlExecutionHosts">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="currentText">
            <string>localhost</string>
           </property>
           <item>
            <property name="text">
             <string>localhost</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbRR2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Remote dir:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leRemoteDirectory">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Given path has to exist. If left empty: unique directory will be created.</string>
           </property>
           <property name="placeholderText">
            <string>(auto create)</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSelectRemoteDirectory">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnResume">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Resume</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnUpload">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Upload to remote location</string>
           </property>
           <property name="text">
            <string>Up</string>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/resources/upload.png</normalon>
            </iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDownload">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Download from remote location</string>
           </property>
           <property name="text">
            <string>Down</string>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/resources/download.png</normalon>
            </iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRemoveRemote">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Remove remote</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="inputTab">
        <attribute name="title">
         <string>Input</string>
        </attribute>
        <layout class="QHBoxLayout" name="inputTabLayout"/>
       </widget>
       <widget class="QWidget" name="runTab">
        <attribute name="title">
         <string>Run</string>
        </attribute>
        <layout class="QVBoxLayout" name="runTabLayout"/>
       </widget>
       <widget class="QWidget" name="outputTab">
        <attribute name="title">
         <string>Output</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QScrollArea" name="scrollArea_2">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="outputContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>1434</width>
              <height>782</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QSplitter" name="splitter">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <widget class="QTreeView" name="resultsToC">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                  <horstretch>1</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
               <widget class="QGroupBox" name="resultElementDetails">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                  <horstretch>4</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="title">
                 <string>Result element details</string>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnRun">
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnKill">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Kill</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDisconnect">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Disconnect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnParaview">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Start ParaView in execution directory</string>
         </property>
         <property name="text">
          <string>ParaView</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnClean">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Clean OpenFOAM case data in execution directory</string>
         </property>
         <property name="text">
          <string>Clean</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnWriteNow">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Trigger immediate time step output in currently running OpenFOAM solver</string>
         </property>
         <property name="text">
          <string>Write now</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnWriteNowAndStop">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Trigger immediate time step output and subsequent stop of currently running OpenFOAM solver</string>
         </property>
         <property name="text">
          <string>Write+Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnShell">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Shell...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
